<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStaff</class>
 <widget class="QMainWindow" name="AddStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_label_PersonalInfo_frame">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Personal Information</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>First Name :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Last Name :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>ID :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Phone :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Email : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Sex :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Birth Date :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Username :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_Fname">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maxLength">
               <number>20</number>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>First Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Lname">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maxLength">
               <number>20</number>
              </property>
              <property name="placeholderText">
               <string>Last Name </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Id">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="inputMask">
               <string>000000000</string>
              </property>
              <property name="maxLength">
               <number>9</number>
              </property>
              <property name="placeholderText">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Phone">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="inputMask">
               <string>00000000000</string>
              </property>
              <property name="maxLength">
               <number>11</number>
              </property>
              <property name="placeholderText">
               <string>Phone Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Email">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maxLength">
               <number>60</number>
              </property>
              <property name="placeholderText">
               <string>E_Mail Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_sex">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>28</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>Male</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Female</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_BirthDat">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="inputMask">
               <string>0000/00/00</string>
              </property>
              <property name="placeholderText">
               <string>Birth Date</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_UserName">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maxLength">
               <number>50</number>
              </property>
              <property name="placeholderText">
               <string>Staff's Username</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_OtherInfo_frame">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Staff Information</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Department :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Type :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Helvetica CE 35 Thin</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Workshift :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QComboBox" name="comboBox_department">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>190</width>
                <height>28</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Research Labratory</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Security</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Academic Affairs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fanancial Affairs</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_type">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>190</width>
                <height>28</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Administrative &amp; Support</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Faculty Member</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_workshift">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>190</width>
                <height>28</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Day Shift</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Night Shift</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Close">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>115</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_top_addstaff">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>50</y>
             <width>106</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>  Add Staff</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_top_addstaff_image">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>0</y>
             <width>48</width>
             <height>48</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_seperator">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_saveInfo">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save Information</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
